src/chain/chain_accumulator.rs

```diff
@@ -293,12 +294,24 @@ mod test {
         ))
     }
 
-    fn random_section_info_sig_payload(rng: &mut MainRng) -> EventSigPayload {
+    fn random_section_info_sig_payload(
+        rng: &mut MainRng,
+    ) -> (EventSigPayload, RealBlsEventSigPayload, RealBlsPublicKeySet) {
```

Comment:
maybe worth to make this triple as a struct?

Reply:
Only a triple because we have a fake and real, will become back a pair when there is only the real one. (i.e bls_emu gone)